### PR TITLE
Core: Add --onboard-wallet-app-icon-background and --onboard-modal-background

### PR DESCRIPTION
**Pull request by ReflectiveChimp**
### Description
- Add `--onboard-wallet-app-icon-background-{color}` to allow setting wallet icon background color.
- Add `--onboard-modal-background` to allow setting modal background color.

image
`--onboard-modal-background` is needed to remove this white dotted line that appears if you change `--onboard-main-scroll-container-background` to something other than white.

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/core/src/views/shared/WalletAppBadge.svelte [hunk 1]
```diff
@@ -78,27 +78,27 @@
   }
 
   .background-gray {
-    background: var(--onboard-gray-500, var(--gray-500));
+    background: var(--onboard-wallet-app-icon-background, var(--onboard-gray-500, var(--gray-500)));
```
Comment (Adamj1232) [inline packages/core/src/views/shared/WalletAppBadge.svelte:81]:
```suggestion
    background: var(--onboard-wallet-app-icon-background-gray, var(--onboard-gray-500, var(--gray-500)));
```
We will want to make each more independent as far as the variable name is concerned. I think post-fixing the variable with the class color type can achieve this

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Hey @ReflectiveChimp! Thanks for the PR!
There was a release today so the base develop branch and version will need an update and a variable naming comment below

**Comment by ReflectiveChimp** [comment]
@Adamj1232 No problem, updated!

I've bumped core version to 2.3.3 - let me now if you need changed.

File: packages/core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.3.2",
+  "version": "2.3.3",
```
[the remark this answers is not in the document]
Comment (ReflectiveChimp) [reply packages/core/package.json]: Merged upstream and updated core version to 2.4.0-alpha.2

**Comment by Adamj1232** [comment]
> @Adamj1232 No problem, updated!
> 
> I've bumped core version to 2.3.3 - let me now if you need changed.

I just realized you are with Beefy. Been using your platform for a while now. Really appreciate your team and product. Your team was one of the main reasons I decided to work full time in the blockchain space. I have worked on a couple of projects with Terry and am in your discord server. If you need any support feel free to reach out. Ill get your requests in development ASAP

**Review by Adamj1232 — APPROVED** [review]

**Pull request merged**